frontend/apps/cloudstorage/ftp.lua

```diff
@@ -52,9 +52,9 @@ function Ftp:downloadFile(item, address, user, pass, path, close)
 end
 
 function Ftp:config(item, callback)
-    local text_info = "FTP address must be in the format ftp://example.domain.com\n"..
+    local text_info = _("FTP address must be in the format ftp://example.domain.com\n"..
```

Comment:
Let's make it a little bit nicer to read too. :-)
```suggestion
    local text_info = _([[
The FTP address must be in the following format:
ftp://example.domain.com

An IP address is also supported, for example:
ftp://10.10.10.1

Username and password are optional.]])
```